### DISCUSSION TITLE
How to change color scheme?

### DISCUSSION THREAD

**Discussion by Yp-pro** [Eloston/ungoogled-chromium#2906] [Q&A]
Hello everyone,

Could you guys say how to change the color scheme please? I mean this color and other stuff (if possible):

![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/156006482/51df7bb5-375e-44a6-8c84-4b4242b4be77)

Thanks

(Windows 10, Chromium Version 125.0.6422.141 (Official Build, ungoogled-chromium) (64-bit))

**Comment by PF4Public** [comment]
`chrome://new-tab-page/` is it what you're looking for?

**Reply by Yp-pro** [reply]
> `chrome://new-tab-page/` is it what you're looking for?

Not really. Something like user.css where all ui colors etc

**Reply by PF4Public** [reply]
Then I doubt something like this exists in Chromium. But let's wait, maybe someone knows better.

**Reply by Qubik65536** [reply]
> Then I doubt something like this exists in Chromium. But let's wait, maybe someone knows better.

![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/72877496/ac487c0c-79bb-499a-a081-85e94bfef8db)

In Thorium, there is such a thing (`Customize` in the bottom right corner) to change the color scheme. I can confirm that Vanilla Chromium has it, but Ungoogled-Chromium doesn't. One of the way, that worked on macOS, is change it using Vanilla Chromium and replace the installation, but this is tbh not a good workaround...

**Reply by PF4Public** [reply]
> Ungoogled-Chromium doesn't

It is exactly `chrome://new-tab-page/` that I already mentioned.

**Reply by Qubik65536** [reply]
> > Ungoogled-Chromium doesn't
> 
> It is exactly `chrome://new-tab-page/` that I already mentioned.

Oh I see... The one that shows when adding a tab is not exactly `chrome://new-tab-page/`... My bad.

**Comment by rmdhnreza** [comment]
You can also change this color from `chrome://settings/manageProfile`

**Reply by PF4Public** [reply]
It is even less obscure compared to `chrome://new-tab-page/` 👍🏻
